.travis.yml

```diff
@@ -22,6 +23,12 @@ matrix:
       env: DJANGO="Django<1.11"
     - python: "3.3"
       env: DJANGO="Django<1.12"
+    - python: "3.6"
+      env: DJANGO="Django<1.9"
+    - python: "3.6"
+      env: DJANGO="Django<1.10"
+    - python: "3.6"
+      env: DJANGO="Django<1.11"
```

Comment:
Pretty sure I got this right.

Django 1.11 is the first version to support Python 3.6--none of the other Django versions do. So I added exclusions for those combinations.